### PR TITLE
Fixes borg grippers not being able to use drop hotkey (Q) to drop items

### PR DESCRIPTION
**Pull request by JohnWildkins**
title

### CODE REVIEW

**Comment by JohnWildkins** [comment]
!bugfix

**Comment by JohnWildkins** [comment]
!review

**Review by Geevies — APPROVED** [review]
gamer

**Review by NonQueueingMatt — APPROVED** [review]

File: html/changelogs/johnwildkins-gripper.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+author: JohnWildkins
+
+delete-after: True
+
+changes:
+  - bugfix: "Drop hotkey (Q) should not properly work to drop things held by robotic grippers."
```
Comment (mikomyazaki) [inline html/changelogs/johnwildkins-gripper.yml:6]:
```suggestion
  - bugfix: "Drop hotkey (Q) should now properly work to drop things held by robotic grippers."
```

**Review by mikomyazaki — CHANGES_REQUESTED** [review]

File: html/changelogs/johnwildkins-gripper.yml (hunk 1 above)
Comment (NonQueueingMatt) [reply html/changelogs/johnwildkins-gripper.yml:6]: lmao

**Pull request merged**